lerobot/configs/robot/myarm.yaml

```diff
@@ -0,0 +1,59 @@
+# [MyArm Robot Arm Tooling](https://github.com/apockill/acton_ai)
```

Comment:
TODO: Test and validate a policy, put a video here to show how it all came together

Reply:
Okay, so here's [a policy](https://photos.app.goo.gl/49igB19tp3ktqC8s9) in action that's not quite working... It was supposed to stack the cubes haha. However, it's definitely "working" in that the whole implementation works- teleop, data recording, training, and finally playback has been tested so far. I will add further PR's to teach how to train better policies in the near future :)

Here's a [teleop example](https://photos.google.com/photo/AF1QipNJ-mpbkLK02lrHncGCyf4Rn2lSoz7z1runM7Lc).